index.bs

```diff
@@ -342,37 +342,37 @@ Each {{WebSocket}} object has an associated <dfn>url</dfn> (a [=URL record=]).
      fails, it triggers the [=fail the WebSocket connection=] algorithm, which
      then invokes the [=close the WebSocket connection=] algorithm, which then
      establishes that [=the WebSocket connection is closed=], which fires the
-     <a event><code>close</code></a> event <a href="#closeWebSocket">as described below</a>.
+     <a event>`close`</a> event <a href="#closeWebSocket">as described below</a>.
  1. Return a new {{WebSocket}} object whose [=url=] is |urlRecord|.
 
 </div>
 
 <hr>
 
-The <dfn attribute for=WebSocket><code>url</code></dfn> attribute's getter must return this
+The <dfn attribute for=WebSocket>`url`</dfn> attribute's getter must return this
```

Comment:
These backticks should not be necessary; `<code>` gets automatically added for attributes and methods and similar.